packages/flutter_driver/lib/src/driver/driver.dart

```diff
@@ -157,10 +157,10 @@ abstract class FlutterDriver {
   }
 
   /// Getter of appIsolate
-  VMIsolate get appIsolate;
+  VMIsolate get appIsolate => throw UnimplementedError();
```

Comment:
What is the rationale for throwing versus abstract? Just curious...

Reply:
Discussed with Jonah offline.